docs/sources/reference/components/otelcol/otelcol.exporter.datadog.md

```diff
@@ -0,0 +1,324 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.exporter.datadog/
+description: Learn about otelcol.exporter.datadog
+labels:
+  stage: community
```

Comment:
I don't think that we should have community as a label here, you can remove the labels and add aliases instead (like in the other otel docs)